### PR TITLE
Tqct 1058 fix warnings

### PR DESCRIPTION
**Pull request by alubenskyi**

### CODE REVIEW

**Review by AlexArcher — APPROVED** [review]

File: spec/translation_quality/checks/insertable_consistency.yaml [hunk 1]
```diff
@@ -25,6 +25,15 @@ components:
                       - INSERTABLE_DELETED
                     description: check subtype code
                     example: "INSERTABLE_ADDED"
+                  severityLevelCode:
+                    type: string
```
Comment (rischenko) [inline spec/translation_quality/checks/insertable_consistency.yaml:29]: ref to tqc_common#severityLevelCodeEnum ?

File: spec/translation_quality/checks/insertable_consistency.yaml [hunk 2]
```diff
@@ -45,12 +54,23 @@ components:
...
+                    nullable: true
                     type: string
                     enum:
                       - INSERTABLE_ADDED
                       - INSERTABLE_DELETED
                     description: check subtype code
                     example: "INSERTABLE_ADDED"
+                  severityLevelCode:
+                    nullable: true
+                    type: string
```
Comment (rischenko) [inline spec/translation_quality/checks/insertable_consistency.yaml:66]: ref?

File: spec/translation_quality/checks/non_translatable_text.yaml [hunk 3]
```diff
@@ -55,6 +67,16 @@ components:
                       - PLACEABLE_FORMAT
                     description: check subtype code
                     example: "PLACEABLE_ADDED"
+                  severityLevelCode:
+                    nullable: true
+                    type: string
+                    enum:
```
Comment (rischenko) [inline spec/translation_quality/checks/non_translatable_text.yaml:73]: ref

File: spec/translation_quality/checks/number_consistency.yaml [hunk 4]
```diff
@@ -45,6 +45,15 @@ components:
                       - NUMBER_INCORRECT_CONVERSION
                     description: check subtype code
                     example: "NUMBER_ADDED"
+                  severityLevelCode:
+                    type: string
+                    enum:
```
Comment (rischenko) [inline spec/translation_quality/checks/number_consistency.yaml:50]: ref

File: spec/translation_quality/checks/number_consistency.yaml [hunk 5]
```diff
@@ -67,12 +76,23 @@ components:
...
                     enum:
                       - NUMBER_ADDED
                       - NUMBER_DELETED
                       - NUMBER_INCORRECT_CONVERSION
                     description: check subtype code
                     example: "NUMBER_ADDED"
+                  severityLevelCode:
+                    nullable: true
+                    type: string
+                    enum:
```
Comment (rischenko) [inline spec/translation_quality/checks/number_consistency.yaml:89]: ref

File: spec/translation_quality/checks/regex_check.yaml [hunk 6]
```diff
@@ -49,7 +49,10 @@ components:
               description: check type code
               example: "REGEX_CHECK"
             details:
-              - ref: '../tqc_common.yaml#/components/schemas/regexRuleWithRuleUid'
+              type: object
+              nullable: true
+              oneOf:
```
Comment (rischenko) [inline spec/translation_quality/checks/regex_check.yaml:54]: do we need oneOf?

File: spec/translation_quality/checks/segment_completeness.yaml [hunk 7]
```diff
@@ -54,6 +54,15 @@ components:
                       - TARGET_IS_MUCH_LONGER_THAN_SOURCE
                     description: check subtype code
                     example: "TARGET_IS_EMPTY"
+                  severityLevelCode:
+                    type: string
+                    enum:
```
Comment (rischenko) [inline spec/translation_quality/checks/segment_completeness.yaml:59]: ref

File: spec/translation_quality/checks/segment_completeness.yaml [hunk 8]
```diff
@@ -75,13 +84,24 @@ components:
...
                     enum:
                       - TARGET_IS_EMPTY
                       - TARGET_IS_MUCH_SHORTER_THAN_SOURCE
                       - TARGET_IS_MUCH_LONGER_THAN_SOURCE
                     description: check subtype code
                     example: "TARGET_IS_EMPTY"
+                  severityLevelCode:
+                    nullable: true
+                    type: string
+                    enum:
```
Comment (rischenko) [inline spec/translation_quality/checks/segment_completeness.yaml:98]: ref

File: spec/translation_quality/checks/tag_consistency.yaml [hunk 9]
```diff
@@ -41,19 +52,31 @@ components:
...
                       - TAG_ADDED
                       - TAG_DELETED
                       - SPACES_AROUND_TAGS
                       - TAG_ORDER_IS_CHANGED
                     description: check subtype code
                     example: "TAG_ADDED"
+                  severityLevelCode:
+                    nullable: true
+                    type: string
+                    enum:
```
Comment (rischenko) [inline spec/translation_quality/checks/tag_consistency.yaml:73]: ref

File: spec/translation_quality/accounts_result.yaml [hunk 10]
```diff
@@ -2590,10 +2992,18 @@ x-paths:
                                                       - SAME_TRANSLATION_DIFFERENT_SOURCES
                                                     description: check subtype code
                                                     example: SAME_SOURCE_DIFFERENT_TRANSLATIONS
+                                                  severityLevelCode:
+                                                    type: string
+                                                    enum:
```
Comment (rischenko) [inline spec/translation_quality/accounts_result.yaml:2997]: ref

File: spec/translation_quality/checks/target_source_consistency.yaml [hunk 11]
```diff
@@ -79,6 +87,15 @@ components:
                       - SAME_TRANSLATION_DIFFERENT_SOURCES
                     description: check subtype code
                     example: "SAME_SOURCE_DIFFERENT_TRANSLATIONS"
+                  severityLevelCode:
+                    type: string
+                    enum:
```
Comment (rischenko) [inline spec/translation_quality/checks/target_source_consistency.yaml:92]: ref

File: spec/translation_quality/checks/target_source_consistency.yaml [hunk 12]
```diff
@@ -52,6 +51,15 @@ components:
...
```
Comment (rischenko) [inline spec/translation_quality/checks/target_source_consistency.yaml:56]: ref

**Review by rischenko — APPROVED** [review]

File: spec/translation_quality/checks/insertable_consistency.yaml (hunk 2 above)
Comment (alubenskyi) [reply spec/translation_quality/checks/insertable_consistency.yaml]: done

File: spec/translation_quality/checks/non_translatable_text.yaml [hunk 13]
```diff
@@ -20,13 +20,24 @@ components:
...
                       - PLACEABLE_ADDED
                       - PLACEABLE_DELETED
                       - SPACES_AROUND_PLACEABLE
                       - PLACEABLE_FORMAT
                     description: check subtype code
                     example: "PLACEABLE_ADDED"
+                  severityLevelCode:
+                    nullable: true
+                    type: string
+                    enum:
```
[the remark this answers is not in the document]
Comment (alubenskyi) [reply spec/translation_quality/checks/non_translatable_text.yaml]: done

File: spec/translation_quality/checks/non_translatable_text.yaml (hunk 3 above)
Comment (alubenskyi) [reply spec/translation_quality/checks/non_translatable_text.yaml]: done

File: spec/translation_quality/checks/number_consistency.yaml (hunk 4 above)
Comment (alubenskyi) [reply spec/translation_quality/checks/number_consistency.yaml]: done

File: spec/translation_quality/checks/number_consistency.yaml (hunk 5 above)
Comment (alubenskyi) [reply spec/translation_quality/checks/number_consistency.yaml]: done

File: spec/translation_quality/checks/regex_check.yaml (hunk 6 above)
Comment (alubenskyi) [reply spec/translation_quality/checks/regex_check.yaml]: removed

File: spec/translation_quality/checks/segment_completeness.yaml (hunk 7 above)
Comment (alubenskyi) [reply spec/translation_quality/checks/segment_completeness.yaml]: done

File: spec/translation_quality/checks/tag_consistency.yaml [hunk 14]
```diff
@@ -26,6 +27,16 @@ components:
...
```
[the remark this answers is not in the document]
Comment (alubenskyi) [reply spec/translation_quality/checks/tag_consistency.yaml]: done

File: spec/translation_quality/checks/tag_consistency.yaml (hunk 9 above)
Comment (alubenskyi) [reply spec/translation_quality/checks/tag_consistency.yaml]: done

File: spec/translation_quality/accounts_result.yaml (hunk 10 above)
Comment (alubenskyi) [reply spec/translation_quality/accounts_result.yaml]: done

File: spec/translation_quality/accounts_result.yaml [hunk 15]
```diff
@@ -2540,11 +2935,20 @@ x-paths:
                                                       - TAG_ORDER_IS_CHANGED
                                                     description: check subtype code
                                                     example: TAG_ADDED
+                                                  severityLevelCode:
+                                                    nullable: true
+                                                    type: string
+                                                    enum:
```
[the remark this answers is not in the document]
Comment (alubenskyi) [reply spec/translation_quality/accounts_result.yaml]: done

File: spec/translation_quality/checks/target_source_consistency.yaml (hunk 11 above)
Comment (alubenskyi) [reply spec/translation_quality/checks/target_source_consistency.yaml]: done

File: spec/translation_quality/checks/target_source_consistency.yaml (hunk 12 above)
Comment (alubenskyi) [reply spec/translation_quality/checks/target_source_consistency.yaml]: done

**Pull request merged**